modules/webgl/src/adapter/converters/device-parameters.ts

```diff
@@ -278,9 +278,17 @@ export function setDeviceParameters(device: Device, parameters: Parameters) {
   // },
 
   // COLOR STATE
+  if (parameters.blend) {
+    gl.enable(GL.BLEND);
+  } else if (parameters.blend === false) {
+    gl.disable(GL.BLEND);
+  }
 
   if (parameters.blendColorOperation || parameters.blendAlphaOperation) {
-    gl.enable(GL.BLEND);
+    if (parameters.blend === undefined) {
```

Comment:
This auto-enabling seems subtle. 
- Is there any reason not to require `blend: true` to be set and remove the auto-enable? 
- If some good backwards compatibility reason then maybe add a comment explaining that so that when we invariably refactor this next time, we don't break things in deck without knowing?